osidb/models.py

```diff
@@ -1775,15 +1775,18 @@ def _validate_historical_affectedness_resolution(self):
                 or self.resolution != old_affect.resolution
             ):
                 raise ValidationError(
-                    f"{self.resolution} is not a valid resolution for {self.affectedness}."
+                    f"Affect ({self.uuid}) for {self.ps_module}/{self.ps_component} has an invalid "
+                    f"resolution/affectedness combination: {self.resolution} is not a valid resolution "
+                    f"for {self.affectedness}."
```

Comment:
The usual order to refer to the affect status is in order of first `affectedness` and then `resolution`. This way people are used to see it in the old tooling and I think the opposite order may be confusing for them. Eventually we should merge it into a single `status` value but at least for the time being it would be better to stick with the current order.